CharlieBackend.Panel/Services/SecretaryService.cs

```diff
@@ -0,0 +1,56 @@
+﻿using CharlieBackend.Panel.Models.Secretary;
+using CharlieBackend.Panel.Services.Interfaces;
+using CharlieBackend.Panel.Utils.Interfaces;
+using Microsoft.Extensions.Options;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace CharlieBackend.Panel.Services
+{
+    public class SecretaryService : ISecretaryService
+    {
+        private readonly IApiUtil _apiUtil;
+        private readonly SecretariesApiEndpoints _secretariesApiEndpoints;        
+
+        public SecretaryService(IApiUtil apiUtil, IOptions<ApplicationSettings> options)
+        {
+            _apiUtil = apiUtil;
+            _secretariesApiEndpoints = options.Value.Urls.ApiEndpoints.Secretaries;            
+        }
+
+        public async Task<bool> DisableSecretaryAsync(long id)
+        {
+            var disableSecretaryEndpoint = string
+                .Format(_secretariesApiEndpoints.DisableSecretaryEndpoint, id);
+
+            return await _apiUtil.DeleteAsync<bool>(disableSecretaryEndpoint);
+        }
+
+        public async Task<bool> EnableSecretaryAsync(long id)
+        {
+            var enableSecretaryEndpoint = string
+                .Format(_secretariesApiEndpoints.EnableSecretaryEndpoint, id);
+
+            return await _apiUtil.EnableAsync<bool>(enableSecretaryEndpoint);
+        }
+
+        public async Task<IList<SecretaryViewModel>> GetAllSecretariesAsync()
+        {
+            var getAllSecretariesEndpoint = _secretariesApiEndpoints.GetAllSecretariesEndpoint;
```

Comment:
please investigate GetAllSecretariesEndpoint and set is active there

Reply:
Was investigated. Added property.

Reply:
Was investigated and added property.